pkg/controller/template/render.go

```diff
@@ -359,6 +360,13 @@ func cloudProvider(cfg RenderConfig) (interface{}, error) {
 	return "", nil
 }
 
+func cloudConfigFlag(cfg RenderConfig) (interface{}, error) {
```

Comment:
I don't quite understand this func? can you add comments explaining why this is here?

Reply:
+1 to comment. I believe this is just a template function, so whenever a template gets processed, if you wrote cloudConfigFlag, it calls this func and renders the result.

Reply:
added the docs.

Reply:
thanks!